tests/Doctrine/Tests/DBAL/Platforms/AbstractMySQLPlatformTestCase.php

```diff
@@ -329,6 +329,32 @@ public function testAlterTableAddPrimaryKey()
     /**
      * @group DBAL-464
      */
+    public function testAlterPrimaryKeyWithAutoincrementColumn()
+    {
+        $table = new Table("drop_primary_key");
+        $table->addColumn('id', 'integer', array('primary' => true, 'autoincrement' => true));
```

Comment:
@deeky666 can do, but the same happens in the original DBAL-464 fix: https://github.com/doctrine/dbal/blob/master/tests/Doctrine/Tests/DBAL/Platforms/AbstractMySQLPlatformTestCase.php#L342
Do you want me to update both?